modules/copy-post.php

```diff
@@ -32,6 +32,7 @@ public function __construct() {
 
 		if ( ! empty( $_GET['jetpack-copy'] ) && 'post-new.php' === $GLOBALS['pagenow'] ) {
 			add_action( 'wp_insert_post', array( $this, 'update_post_data' ), 10, 3 );
+			add_filter( 'pre_option_default_post_format', array( $this, 'shortcircuit_default_post_format' ) );
```

Comment:
Could we use `__return_empty_string` here instead of creating an extra function?

Reply:
Ha! Well spotted, I had no idea we had that. I was even thinking, "gee, too bad I can't return false, I could have just used `__return_false`" 😜Will do!